app/views/govuk_publishing_components/components/_cookie_banner.html.erb

```diff
@@ -0,0 +1,8 @@
+<% message ||= capture do %>
+  GOV.UK uses cookies to make the site simpler. <a class="govuk-link" href="https://www.gov.uk/help/cookies" data-module="track-click" data-track-category="cookieBanner" data-track-action="Cookie banner clicked">Find out more about cookies</a> or <a class="govuk-link" href="#" data-hide-cookie-banner="true" data-module="track-click" data-track-category="cookieBanner" data-track-action="Cookie banner hidden">hide this message</a>
+<% end %>
+<% id ||= 'global-cookie-message' %>
```

Comment:
Could this line of code be included with the lines above? e.g.

```
<% 
  message ||= capture do
    GOV.UK uses cookies...
  end

  id ||= 'global-cookie-message'
%>
```

Reply:
I don't think it works this day as the message includes html tags. Will swap them and use a single block.

Reply:
Oh, of course. Never mind then.